.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -56,10 +56,10 @@ schedules:
 # Global variables
 variables:
   buildConfiguration: Release
-  dotnetCoreSdkLatestVersion: 9.0.203
+  dotnetCoreSdkLatestVersion: 10.0.100-preview.5.25277.114
   # This is required until we're out of rc.
   # After GA we can do a find and replace and get rid of this
-  dotnetCoreSdkLatestVersionShort: 9.0.203
+  dotnetCoreSdkLatestVersionShort: 10.0.100-preview.5.25277.114
```

Comment:
I think this needs to be `10.0.100-preview.5` or maybe `10.0.100-preview` otherwise you'll get docker pull errors